controllers/users.js

```diff
@@ -284,6 +291,144 @@ class Users {
       }))
       .catch(err => next(err));
   }
+
+  /**
+   * A user can follow another user
+   * @param {object} req
+   * @param {object} res
+   * @return {object} res
+   * */
+  static async follow(req, res) {
+    const { username } = req.params;
+    const { id } = req.user;
+    try {
+      const userExists = await User.findOne({
+        where: {
+          username,
+          id: { [Sequelize.Op.ne]: id },
+        },
+        attributes: ['id', 'username', 'bio', 'image', 'followers']
+      });
+
+      if (!userExists) {
+        return res.status(404).json({
+          status: res.statusCode,
+          message: 'the user you want to follow doesn\'t exist',
```

Comment:
@avpaul please fix this type in the message: spelling of "doesn\'t" it should be "doesn't"

Reply:
Sorry, @aaronsekisambu I don't see any typo here. The slash there is for escaping the single quote(apostrophe)